README.md

```diff
@@ -1,65 +1,76 @@
 # Giraffe
 
-A React-based visualization library powering the data visualizations in [InfluxDB 2.0](https://github.com/influxdata/influxdb/) UI.
+A React-based visualization library powering the data visualizations in the [InfluxDB 2.0](https://github.com/influxdata/influxdb/) UI.
 
-**This library is currently in pre-beta**
+(*This library is currently in pre-beta.*)
```

Comment:
what is `pre-beta`? `alpha`?

Reply:
I had the same question. Would `alpha` be accurate, @TCL735 @desa ?

Reply:
Previously, the description was a "pre-alpha" state. It was never updated even though much work and progress had been made. It was close to being in beta. And that's where it is today: close to being in beta. Rather than saying that or saying beta, "pre-alpha" was edited to "pre-beta". We can call it whatever makes sense. It is about to be in beta.

Reply:
Okay! Thanks—